simple-example/model_definition.yaml

```diff
@@ -17,17 +17,17 @@ spec:
         desc: The number of items in the sequence.
         type: int
         default: 20
-        min: 5
+        min: 2
         
       - name: SEQUENCE_F0
         title: F0 start number
-        desc: The initial start value for the sequence.
+        desc: The initial start value for the sequence. Must be an integer.
```

Comment:
Do we need this addition if the type is set to int? If you think we do then it should probably also be added to the sequence length description too

Reply:
I was not sure about this either. Should the description restate the settings in the model definition? Should we try to extend the description in the front end? Or should we leave it at the basic description?

I stated it here specifically because in theory a Fibanacci sequence can take a float value in F0 and F1 - however this particular implementation doesn't handle that case.

Reply:
I'll remove it for now.